desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -130,10 +147,64 @@ void TabDiveInformation::divesChanged(dive_trip *trip, const QVector<dive *> &di
 	case DiveField::WATER_TEMP:
 		ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 		break;
+	case DiveField::ATM_PRESS:
+		ui->atmPressVal->setText(ui->atmPressVal->text().sprintf("%d",current_dive->surface_pressure.mbar));
+		break;
 	case DiveField::DATETIME:
 		updateWhen();
 		break;
 	default:
 		break;
 	}
 }
+
+void TabDiveInformation::updateTextBox(int i)
```

Comment:
Why not something more descriptive, such as `bool typeChanged` or similar?

Reply:
The purpose of the function is to consider the index of the combobox, store an appropriate number in the text box and save it in the dive structure. A *change* in index is only one of the triggers for this function. The more important trigger is editing of the text box. I cannot really think of a more descriptive name for the function. "typeChanged" would be less accurate.

Reply:
I meant the parameter, not the function. :)

Reply:
I think the #defines that I used make it more clear.